tools/autograd/derivatives.yaml

```diff
@@ -138,7 +138,7 @@
 - name: cat(TensorList tensors, int64_t dim)
   tensors: cat_tensors_backward(grad, to_arg_sizes(tensors, dim), dim)
 
-- name: cauchy(Tensor self, double median, double sigma, Generator generator)
+- name: cauchy_(Tensor self, double median, double sigma, Generator generator)
```

Comment:
ok, now I'm confused about when I need to write an inplace function name here.  I.e. there is no add_ here, but there is a cauchy_.  Is it if there is _only_ an inplace function?  Below it says "In-place functions wil use the out-of-place derivative definition if there is no in-place specific derivative" -- that's ambiguous about whether I can `cachy(` here or not.  It's an out-of-place definition, but is it valid?  Can you add a comment to the top describing a rule for people to use?